opentelemetry-sdk/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Update sampling result names
   ([#1128](https://github.com/open-telemetry/opentelemetry-python/pull/1128))
+- Add support for OTEL_BSP_X environment variables
```

Comment:
It would be helpful to have the variables listed out here.

Reply:
Fixed